builder/packages_image.go

```diff
@@ -150,16 +149,6 @@ func (p *PackagesImageBuilder) NewDockerPopulator(roleManifest *model.RoleManife
 			return err
 		}
 		defer os.RemoveAll(specDir)
-		configStore := configstore.NewConfigStoreBuilder(
-			configstore.JSONProvider,
-			lightManifestPath,
-			darkManifestPath,
-			specDir,
-		)
-
-		if err = configStore.WriteBaseConfig(roleManifest); err != nil {
-			return fmt.Errorf("Error writing base config: %s", err.Error())
-		}
 		walker := &tarWalker{stream: tarWriter, root: specDir, prefix: "specs"}
```

Comment:
The `specDir` is empty, we shouldn't create it, nor show we walk it.  This also means we need to remove it from `Dockerfile-packages`.